tools/shared/src/main/scala/org/scalajs/core/tools/optimizer/Emitter.scala

```diff
@@ -47,6 +47,11 @@ final class Emitter(semantics: Semantics, outputMode: OutputMode) {
     emitPostlude(builder, logger)
   }
 
+  def emitCustomHeader(customHeader: String, builder: JSFileBuilder): Unit = {
+    if(customHeader != "")
+      builder.addLine(customHeader)
```

Comment:
Space after `if`.
Also, `builder.addLine` should be called several times if there are `\n`'s in `customHeader`. `addLine` is not equipped to deal with new lines in its parameter, especially if the builder creates a source map.